src/components/shared/form_inputs/FunDynamicSubmitButton.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+	<button
+		class="button"
+		:type="buttonType"
+		v-bind="$attrs"
+	>
+		{{ $t( dynamicButtonText ) }}
+	</button>
+</template>
+
+<script lang="ts">
+import { computed, defineComponent, PropType } from 'vue';
+
+export default defineComponent( {
```

Comment:
You should convert this to setup script as eventually all our components will be in that style. To do that you need to pull the code from the setup() function into the script body (without the return) and add an interface to define the props:

```js
interface Props {
    paymentType: 'PPL'|'UEB'|'MCP'|'SUB';
    buttonType?: 'button'|'submit'|'reset';
}

const props = withDefaults( defineProps<Props>, {
    buttonType: 'button',
} );
```